react/ConditionLayoutProduct.tsx

```diff
@@ -94,7 +99,13 @@ const ConditionLayoutProduct: StorefrontFunctionComponent<Props> = ({
     properties: specificationProperties,
   } = product ?? {}
 
-  const { itemId: selectedItemId } = selectedItem ?? {}
+  const { itemId: selectedItemId, sellers } = selectedItem ?? {}
+  const sellersWithProductAvailable = sellers?.filter(
+    (seller) => seller.commertialOffer.AvailableQuantity > 0
+  )
+  const isProductAvailable =
```

Comment:
This can be moved to the `isProductAvailable` method like you were doing before! It returns if a product is available or not. If a dev wants to know if a product is NOT available, they can change their expectation by passing the `toBe: false` prop to the condition-layout component.

If you have questions or if I need to be clearer, just say it 😁

Reply:
@kaisermann Done. Could check it again, pls?